main.py

```diff
@@ -105,18 +106,19 @@ def evaluate_policy(policy, env, episodes):
     Run the environment env using policy for episodes number of times.
     Return: average rewards per episode.
     """
-    total_reward = 0.0
+    rewards = []
     for _ in range(episodes):
-        state = env.reset()
+        state = np.float32(env.reset())
         is_terminal = False
         while not is_terminal:
             action = policy.get_action(tf.convert_to_tensor([state], dtype=tf.float32))
             # remove the batch_size dimension if batch_size == 1
             action = tf.squeeze(action, [0]).numpy()
             state, reward, is_terminal, _ = env.step(action)
-            total_reward += reward
+            state, reward = np.float32(state), np.float32(reward)
+            rewards.append(reward)
             # env.render()
-    return total_reward / episodes
+    return rewards

```

Comment:
I'd keep this as average rewards

Reply:
list is better, since we will also get std, which means uncertainty.